Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
```

Comment:
I agree. Additionally, I think there could be a space or an extra line when prompted for input to improve readability for the user.

Reply:
This is a great solution to keep the modulo operator consistent with the other operators. I might also add some exception handling with the division and modulo operators to prevent bad inputs from breaking the code.